packages/aila/src/protocol/schema.ts

```diff
@@ -84,29 +80,31 @@ export type MisconceptionsOptional = z.infer<
   typeof MisconceptionsOptionalSchema
 >;
 
-export const QuizQuestionOptionalSchema = z.object({
-  question: z.string().optional(),
-  answers: z.array(z.string()).optional(),
-  distractors: z.array(z.string()).optional(),
-});
+// ********** QUIZ **********
 
-export const QuizQuestionSchema = z.object({
-  question: z.string().describe("The question to be asked in the quiz."),
-  answers: z.array(z.string()).length(1).describe("The correct answer."),
-  distractors: z.array(z.string()).length(2).describe("A set of distractors."),
-});
+export const QUIZ_DESCRIPTIONS = {
+  question: "The question to be asked in the quiz.",
+  answers: "The correct answer. This should be an array of only one item.",
+  distractors: "A set of distractors. This must be an array with two items.",
+} as const as {
```

Comment:
This typing is somewhat annoying but seems to be necessary unless anyone has a better idea.

Reply:
Can you explain what the problem is with the type? As far as I can tell, this is already the type of the literal object in this case

Reply:
To be able to use const we have to also add the { question: string; ...} etc. I tried it without or with Record<string, string> but that was giving me complaints.

Reply:
This seems to be resolved now!

Reply:
Lovely